app_dart/lib/src/model/ci_yaml/target.dart

```diff
@@ -141,6 +141,19 @@ class Target {
   String getBucket() {
     return value.bringup ? 'staging' : 'prod';
   }
+
+  /// Check if this [Target] is benchmark.
+  ///
+  /// keyword benchmark is extracted from the properties.
+  bool isBenchmark() {
```

Comment:
```suggestion
static const String kBenchmarkPropertyKey = 'benchmark';

bool get isBenchmark => getProperties()['benchmark'] == true ?? false;
```

Reply:
Nit on @CaseyHillers comment above: `getProperties()['benchmark']` => `_getTargetProperties()['benchmark']`

As this is only defined on target level.

Reply:
`getProperties` is the properties defined for a target (as its combination of local and platform properties). I would avoid looking at local or platform specific properties separately.